cmd/mdatagen/internal/templates/config.go.tmpl

```diff
@@ -13,7 +13,7 @@ import (
 // MetricConfig provides common config for a particular metric.
 type MetricConfig struct {
 	Enabled bool `mapstructure:"enabled"`
-
+	Interval uint64 `mapstructure:"interval"`
```

Comment:
What does an interval mean for a metric?

Reply:
Apologies for the confusion in the description. This is primarily intended for database scrapers. Since these metrics are queried directly from the database and some of them are resource-intensive, it would be beneficial to allow certain metrics to be scraped less frequently.

Reply:
We should make the generation of this config optional then.

So, do you mean that in the SQL world, every metric is a sql request?

Reply:
I cannot speak for all databases, but let's take MSSQL and PostgreSQL as examples. In the contrib repository, these metrics are collected by sending SQL queries.

For reference:

PostgreSQL: [client.go#L293](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/receiver/postgresqlreceiver/client.go#L293)

MSSQL: [queries.go#L14](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/receiver/sqlserverreceiver/queries.go#L14)

I will update the code to make this behavior optional

Reply:
updated, please take a look, thanks!